Business/DependencyResolvers/Autofac/AutofacBusinessModule.cs

```diff
@@ -1,9 +1,12 @@
 ﻿using Autofac;
-using WebNetSample.DataAccess.Abstract;
-using WebNetSample.DataAccess.Concrete.EntityFramework;
 using WebNetSample.Business.Abstract;
 using WebNetSample.Business.Concrete;
 using WebNetSample.Core.Pagination;
+using System.Reflection;
+using Castle.DynamicProxy;
+using WebNetSample.Core.Utilities.Interceptors;
```

Comment:
It is also a good practice to make order of your namespaces consistent, like making alphabetical order. Let's do it!

Reply:
Can you check my last commit for all change request for this branch?